[BUG] Clarify relationship between default risk score and severity levels (#1722)
elastic/security-docs: docs/detections/rules-ui-create.asciidoc

image::images/severity-mapping-ui.png[]
+
NOTE: For threshold rules, not all source event values can be used for overrides; only the fields that were aggregated over (the `Group by` fields) will contain data. Please also note that overrides are not supported for event correlation rules.

> .. *Default risk score*: A numerical value between 0 and 100 that indicates the risk of events detected by the rule. This setting is set to a default value when you change the *Severity* level, but you can adjust the risk score as needed. General guidelines are:

Reviewer: ```suggestion
.. *Default risk score*: A numerical value between 0 and 100 that indicates the risk of events detected by the rule. It defaults to a preset when you change the *Severity* level, but you can adjust it as needed. General guidelines are:
```

> .. *Default risk score*: A numerical value between 0 and 100 that indicates the risk of events detected by the rule. This setting is set to a default value when you change the *Severity* level, but you can also adjust the risk score as needed. General guidelines are:

Reviewer 2: ```suggestion
.. *Default risk score*: A numerical value between 0 and 100 that indicates the risk of events detected by the rule. This setting is set to a default value when you change the *Severity* level, but you can adjust the risk score as needed. General guidelines are:
```
